store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/util/FederatedStoreUtilTest.java

```diff
@@ -56,71 +54,6 @@
 import static org.mockito.Mockito.mock;
 
 public class FederatedStoreUtilTest {
-    @Test
-    public void shouldGetGraphIds() {
```

Comment:
Doesn't graphIds still need to be tested?

Reply:
This is a test for a Util method which has been deleted.

Reply:
I mean there is a new method called getDeprecatedGraphIds or something and I am wondering if that should be tested here instead

Reply:
It work because lots of test are using it when `FederatedOperation` isn't being used. All these tests will fail if this deprecated method wasn't working right. this method will be removed very soon.